packages/shared-ux/modal/tabbed/src/tabbed_modal.tsx

```diff
@@ -113,6 +113,7 @@ const TabbedModalInner: FC<ITabbedModalInner> = ({
       }}
       style={{ ...(modalWidth ? { width: modalWidth } : {}) }}
       maxWidth={true}
+      aria-labelledby="shareContextModal"
```

Comment:
The `aria-labelledby` should point to an ID on the page. I recommend adding a unique ID to the H1 in the share modal, and referencing that ID here. You can verify the change is working as expected by turning on VoiceOver and listening to the modal being announced when it appears on the screen. The modal should match the H1 "Share this search".

Let me know if you want to review or discuss more!